Tests/SwiftDocCTests/Test Bundles/Snippets.docc/Test-snippets.symbols.json

```diff
@@ -0,0 +1,65 @@
+{
+  "metadata": {
+    "generator": "SwiftPM",
+    "formatVersion": {
+      "major": 0,
+      "minor": 1,
+      "patch": 0
+    }
+  },
+  "relationships": [
+    {
+      "kind": "memberOf",
+      "source": "$snippet__Test.Snippets.MySnippet",
+      "target": "$snippet__Test.Snippets",
+      "targetFallback": null
+    }
+  ],
+  "symbols": [
+    {
+      "isVirtual": true,
+      "docComment": {
+        "lines": [
+          {
+            "text": "Does a foo. "
+          }
+        ]
+      },
+      "snippet": {
+	"language": "swift",
```

Comment:
nit: Looks like the formatting is slightly off here (uses tabs instead of spaces like the rest of the file)

Reply:
Ah, yep! New machine, new vimrc and all that. Thanks